bigquery_etl/cli/view.py

```diff
@@ -121,14 +124,17 @@ def validate(
         sys.exit(1)
 
     # dryrun views
-    ctx.invoke(
-        dryrun,
-        paths=[str(f) for f in view_files],
-        use_cloud_function=use_cloud_function,
-        project=project_id,
-        validate_schemas=validate_schemas,
-        respect_skip=respect_dryrun_skip,
-    )
+    if not skip_dryrun:
+        ctx.invoke(
+            dryrun,
+            paths=[str(f) for f in view_files],
+            use_cloud_function=use_cloud_function,
+            project=project_id,
+            validate_schemas=validate_schemas,
+            respect_skip=respect_dryrun_skip,
+        )
+    else:
+        click.echo("Skip view dryruns.")
```

Comment:
For consistency with the similar message when validating queries.
```suggestion
        click.echo("Dry run skipped for view files.")
```